index.html

```diff
@@ -78,7 +78,8 @@
     <title>CALCULATOR | YOSHITHA</title>
   </head>
 
-  <body class="bg-img">
+  <body class="img">
```

Comment:
This is not for the background image. There is a image without a class add a class for that @bhavy2202